app/models/course.rb

```diff
@@ -82,6 +82,7 @@ class Course < ApplicationRecord
            ->(course) { where(provider_code: course.provider.provider_code) },
            foreign_key: :ucas_course_code,
            primary_key: :course_code,
+           inverse_of: 'course',
```

Comment:
What does this do?

Reply:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/InverseOf